test/rules.json

```diff
@@ -1305,6 +1316,29 @@
       "transformers": [
         "lowercase"
       ]
+    },
+    {
```

Comment:
can we move this before the _VERY LONG RULE MEANT TO TEST DATA TRUNCATION_ ? xD
scrolling 1300 lines isn't fun